kahuna/public/config.js

```diff
@@ -95,6 +95,10 @@ System.config({
     "github:jspm/nodelibs-util@0.1.0": {
       "util": "npm:util@0.10.3"
     },
+    "github:tapmodo/Jcrop@0.9.12": {
```

Comment:
I don't remember adding that manually, but I think it was a requirement somewhere along the line. I can remove it and see if it breaks :)